### PR TITLE
dev-python/openpyxl: version bump 2.5.8

### PR DESCRIPTION
**Pull request by kamiyaa**
Package-Manager: Portage-2.3.49, Repoman-2.3.10
Signed-off-by: Jiayi Zhao <jeff.no.zhao@gmail.com>

### CODE REVIEW

**Comment by mgorny** [comment]
Thanks. I'm going to try to bump its deps and enable Py3.7 everywhere as well.

File: dev-python/openpyxl/openpyxl-2.5.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+DEPEND="
+	test? (
+		${RDEPEND}
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/lxml[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)"
+
+python_test() {
+	pytest -v || die "Testing failed with ${EPYTHON}"
```
Comment (mgorny) [inline dev-python/openpyxl/openpyxl-2.5.8.ebuild]: Also `-vv`, please.

Comment (kamiyaa) [reply dev-python/openpyxl/openpyxl-2.5.8.ebuild]: added `-vv`
Sorry, I don't have a lot of time to test this right now.
Will probably be a while before I can fix it.

**Comment by mgorny** [comment]
Don't worry. I'd suggest that you:

1. Check whether you can reproduce it.
2. If you can reproduce it, try different Python versions. Maybe the package simply doesn't work with py2.7 anymore.
3. Try unpacking the package locally and running the tests manually.

This usually gives some ideas on what could be wrong.

**Pull request closed without merging**